SQL/migrate-2023/V012__increase_tag_size.sql

```diff
@@ -0,0 +1,6 @@
+--
+-- Increases IPC tag size to 20
+--
+
+ALTER TABLE `ss13_characters_ipc_tags` 
+MODIFY COLUMN `serial_number` VARCHAR(20);
```

Comment:
```suggestion
ALTER TABLE `ss13_characters_ipc_tags` MODIFY COLUMN `serial_number` VARCHAR(20);
```
okay can this be one line again?